server/routes/api/stats.py

```diff
@@ -224,3 +224,15 @@ def get_stats_all():
     return Response(json.dumps(dc.get_stats_all(dcids, stat_vars)),
                     200,
                     mimetype="application/json")
+
+
+@bp.route('/api/stats/stat-var-summary', methods=["POST"])
+@cache.cached(timeout=3600 * 24, query_string=True)
```

Comment:
can you check if the cache works for POST? I remember it has issue with memoize, but not sure for "query_string=True"

Reply:
yes, you're right, the cache has problems. I removed the caching.

Reply:
you're right, cache has issues, so I removed the caching.